README.md

```diff
@@ -1,2 +1,242 @@
-# eeajs
-EEA JavaScript libraries.
+# EEA JavaScript libraries - EEAJS
+EEA JavaScript libraries to interact with Pantheon-Quickstart exploring the privacy feature nd deploy contracts.
+
+## Getting Started
```

Comment:
I'd remove this section and add the instruction to clone the repository as the first step in Installing (after pre-requisites)